app/gulp/js/questions.js

```diff
@@ -1,8 +1,14 @@
 // Code to control adding/removing answer choices
 $('body.questions.edit').ready(() => {
+  let num = 10000; // We only support 10000 answers per question, get over it
   $('#add-answer').click((e) => {
+    num -= 1;
     e.preventDefault();
-    $('#wrong-answers').append($('#new-answer').html());
+    let newAnswer = $('#wrong-answers div:first-child').prop('outerHTML');
+    newAnswer = newAnswer.replace(/_[0-9]+_/g, `_${num}_`);
+    newAnswer = newAnswer.replace(/\[[0-9]+\]/g, `[${num}]`);
+    newAnswer = newAnswer.replace(/value="*."/g, 'value=""');
+    $('#wrong-answers').append(newAnswer);
```

Comment:
(If you leave this in, a quick comment saying that it's doing the above might be good.)